liquidation-monitor/src/utils.js

```diff
@@ -0,0 +1,46 @@
+function getAbi(abiName) {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const { abi } = require(`../abi/${abiName}`);
+  return abi;
+}
+
+// Time-stamping + Troubleshooting function
+function ts(input) {
+  const today = new Date();
+  const hours = String(today.getHours()).padStart(2, '0');
+  const minutes = String(today.getMinutes()).padStart(2, '0');
+  const seconds = String(today.getSeconds()).padStart(2, '0');
+  const time = `${hours}:${minutes}:${seconds}`;
+  console.log('TS -', time, input);
+}
+
+// To-do: Replace fetch with axios
+const fetch = require('node-fetch-commonjs');
+
+async function callAPI(url, jsonRequest) {
+  const callPromise = fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(jsonRequest),
+  })
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error('Got non-2XX response from API server.');
+      }
+      return response.json();
+    })
+    .then((responseData) => responseData);
+
+  const responseData = await callPromise.then(
+    (data) => data,
+    (error) => {
+      console.error('Failed to fetch accounts due to error: ', error);
+    },
+  );
```

Comment:
For readability and consistency, we prefer to not mix `then` with `await`.

By convention, we use `await` throughout unless there is a compelling reason to do otherwise.

The following code _should_ be equivalent to the original (this suggestion is untested).
```suggestion
  const response = await fetch(url, {
    method: 'POST',
    body: JSON.stringify(jsonRequest),
  });

  if (!response.ok) {
    console.error('Got non-2XX response from API server.');
  }
  
  let responseData;
  try {
    responseData = await response.json();
  } catch (error) {
    console.error('Failed to fetch accounts due to error: ', error);
  }
```